movie-watchlist.server/Repositories/watchlist/IWatchlistRepo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using movie_watchlist.server.Models;
+
+namespace movie_watchlist.server.Repositories.watchlist
+{
+    public interface IWatchlistRepo
+    {
+        public Task<Watchlist?> CreateWatchlistAsync(string name, string description);
+        public Task<List<Watchlist>> GetWatchlistsAsync();
+        public Task<Watchlist?> GetWatchlistByIdAsync(int watchlistId);
+        public Task<Watchlist?> UpdateWatchlistAsync(int watchlistId, string name, string description, List<int> movieIDs);
+        public Task<bool> DeleteWatchlistAsync(int watchlistId);
```

Comment:
return the deleted Watchlist object? The frontend could implement a "reverse delete, etc with this data